velox/docs/configs.rst

```diff
@@ -179,6 +179,15 @@ sets the update mode to indicate overwriting a partition if exists.
 ``ERROR`` sets the update mode to indicate error throwing if writing
 to an existing partition.
 
+``immutable_partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``bool``
+    * **Default value:** ``false``
+
+Whether new data can be inserted into existing partitions.
```

Comment:
It seems like this property applies only to unpartitioned tables. It would be nice to clarify that especially given that property name suggests that it allows to modify partitioned tables.

Reply:
Updated.